modules/steam.nix

```
@@ -30,11 +30,19 @@ let
   # launch, it therefore needs the CAP_SYS_NICE capability. Bubblewrap
   # can't run a binary with such a capability without being Setuid
   # itself.
-  steam = pkgs.steam.override {
-    buildFHSUserEnv = pkgs.buildFHSUserEnvBubblewrap.override {
-      bubblewrap = "${config.security.wrapperDir}/..";
+  steam = 
+  if (pkgs.lib.functionArgs pkgs.steam.override) ? "buildFHSEnv" then 
```

Yea this doesn't seem to work for unstable. How about:

```suggestion
  if pkgs ? "buildFHSEnv" then 
```

That seems to work on both stable and unstable.

---

Thanks, applied.